rasa/core/tracker_store.py

```diff
@@ -519,6 +520,23 @@ def keys(self) -> Iterable[Text]:
         return [c["sender_id"] for c in self.conversations.find()]
 
 
+def _create_sequence(table_name: Text) -> Sequence:
+    """Creates a sequence object for a specific table name. If using Oracle
```

Comment:
```suggestion
    """Creates a sequence object for a specific table. 
    
    If using Oracle
```

Reply:
why? :P

Reply:
following their example here http://google.github.io/styleguide/pyguide.html#383-functions-and-methods and also our conventions https://www.notion.so/rasa/Python-Code-Conventions-26835ffd72484dd699f28fa693916d2e